### ISSUE TITLE
USER ISSUE: Fish not being updated

### ISSUE THREAD

**Issue by QuantumGO** [StrangeLoopGames/EcoIssues#4653]
**Version:** 0.7.1.2 beta

**Steps to Reproduce:**
1. Catch a fish
2. Hook in the fish

**Expected behavior:**
Get the fish in your inventory

**Actual behavior:**
Doesnt go in your inventory, only after a rejoin

**Comment by VAlex0** [comment]
-> already in this isses #4588

**Comment by MarjieAVolk** [comment]
Closing as duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/4588
